### DISCUSSION TITLE
Freezing before...

### DISCUSSION THREAD

**Discussion by Dimu3yc** [ErinaSugino/Starbound-SBR-Mod#50] [Trouble Shooting] [answered]
Ok, im tried almost everything, and still not working.
POV version is newest
Patches are installed
(also im probably dont understand something in instruction...)
But it still freezing before animation, standing in waiting pose.
Need help.
[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/12619474/starbound.log)
I apologize in advance for my bad english

**Comment by ErinaSugino** [comment, marked as answer]
You have the wrong version of TeratophiliaScenes. The correct one is the file download starting with "SxB_".

Also, it does not seem like you have POV installed at all. You only have a folder named "SxB - HYLOTL POV", which is either installed wrongly or simply contains the images for the *actual* POV mod to use. You do not have POV Redux installed, however.

There is also two more stray folders that don't belong there in the mods folder, one being called "lua" and one being the folder you extracted SBR from. I advice you to remove both from the mods folder to ensure no accidents happen.

**Reply by Dimu3yc** [reply]
Thank you for help.
Also, probably all got broken because of first problem. The last one was not the source of problems
